capa/features/extractors/ida/insn.py

```diff
@@ -12,12 +12,23 @@
 
 import capa.features.extractors.helpers
 import capa.features.extractors.ida.helpers
-from capa.features import MAX_BYTES_FEATURE_SIZE, Bytes, String, Characteristic
+from capa.features import ARCH_X32, ARCH_X64, MAX_BYTES_FEATURE_SIZE, Bytes, String, Characteristic
 from capa.features.insn import Number, Offset, Mnemonic
 
 _file_imports_cache = None
 
 
+def get_arch():
+    # https://reverseengineering.stackexchange.com/a/11398/17194
```

Comment:
I think this would be enough for posts after `Feb 1, 2016`, but this answer is from 2015, which means it is licensed under CC BY-SA 3.0: https://meta.stackexchange.com/questions/271080/the-mit-license-clarity-on-using-code-on-stack-overflow-and-stack-exchange

I think we don't have to worry about the Share-Alike clause as this is an small snippet which has been adapted: https://meta.stackexchange.com/questions/139698/re-using-ideas-or-small-pieces-of-code-from-stackoverflow-com But I think we should add the author's name and profile link as well: https://stackoverflow.blog/2009/06/25/attribution-required